src/aws/cloudwatch-sink.cpp

```diff
@@ -363,16 +370,19 @@ void CloudWatchSink::init_context_info(nlohmann::json& dst,
             throw std::runtime_error(fmt::format("Wrong context_info destination type {}", dst.type_name()));
     }
 
-    for (auto const& itr : context_info)
+    for (auto const& ci_itr : {log.context_info(), context_info, global_context_info})
     {
-        if (!dst.contains(itr.first))
```

Comment:
I don't really understand why this condition was here, what scenario was this supposed to protect against?
(probably doesn't matter anyway now that we changed the code, but I feel like I'm missing something)